Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```
@@ -3621,6 +3621,180 @@ script:
     description: Remove a tag from one or more endpoints.
     execution: false
     name: core-remove-endpoint-tag
+  - name: core-list-users
+    description: Retrieve a list of the current users in your environment.
+    arguments: []
+    outputs:
+    - contextPath: Core.User.user_email
+      description: Email address of the user
+      type: string
+    - contextPath: Core.User.user_first_name
+      description: First name of the user
+      type: string
+    - contextPath: Core.User.user_last_name
+      description: Last name of the user.
+      type: string
+    - contextPath: Core.User.role_name
+      description: Role name associated with the user.
+      type: string
+    - contextPath: Core.User.last_logged_in
+      description: Timestamp of when the user last logged in.
+      type: integer
+    - contextPath: Core.User.user_type
+      description: Type of user.
+      type: string
+    - contextPath: Core.User.groups
+      description: Name of user groups associated with the user, if applicable.
+      type: array
+    - contextPath: Core.User.scope
+      description: Name of scope associated with the user, if applicable.
+      type: array
+  - arguments:
+    - description: |
+        Unique ID of a specific user.
+        User ID could be in the following format: dummy/dummy
```

`could` Is it not always in this format?

---

No, it can be just a "dummy"  without  "/"

---

then let's mention both formats: 
```suggestion
        User ID could be either of the `foo/bar` format, or `somename`
```
change `foo`, `bar` according to the answer to [this question](https://github.com/demisto/content/pull/26211#discussion_r1190185564)